lib/membrane/rtp/ssrc_router.ex

```diff
@@ -325,18 +333,28 @@ defmodule Membrane.RTP.SSRCRouter do
   defp emit_packet_arrival_events(actions, ctx) do
     for action <- actions do
       with {:buffer, {pad, buffer}} <- action do
-        emit_packet_arrival_event(buffer.payload, pad, ctx)
+        emit_packet_arrival_event(buffer, pad, ctx)
       end
     end
   end
 
-  defp emit_packet_arrival_event(payload, pad, ctx) do
+  defp emit_packet_arrival_event(%Buffer{} = buffer, pad, ctx) do
+    packet_size = byte_size(buffer.payload)
+
+    rtp_payload_size =
+      packet_size - buffer.metadata.rtp.padding_size - buffer.metadata.rtp.total_header_size
```

Comment:
That's true only when RTP packet is encrypted. Otherwise, buffer.payload only contains RTP payload 
https://github.com/membraneframework/membrane_rtp_plugin/blob/master/lib/membrane/rtp/packet.ex#L120

I would consider checking against padding_size == 254